src/main.js

```diff
@@ -93,7 +94,16 @@ module.exports = {
       }
     }))
 
-    this.subscriptions.add(atom.commands.add('atom-text-editor', {
+    // Marks each JavaScript editor with a CSS class so that
+    // we can enable commands only for JavaScript editors.
+    this.subscriptions.add(atom.workspace.observeTextEditors((textEditor) => {
+      if (textEditor.getRootScopeDescriptor().scopes
```

Comment:
Can you change this to match the [current checks](https://github.com/Xapphire13/linter-eslint/blob/bd2e81f176f81027a2b94a55a3310bf9b0a4ab97/src/main.js#L51), in fact that check should just be moved to a helper function and called in both places.